seqr/fixtures/clinvar_all_variants.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
a bit nitpicky but this file should be in `clickhouse_search/fixtures`, not `seqr/fixtures`. Moving the file will not change how the fixture setup is done for tests